msaad.go

```diff
@@ -503,6 +503,7 @@ func removeFromGroupList(list []GroupIDU32, i int) []GroupIDU32 {
 
 func (m *MSAAD) syncRoles(roleGroups *cachedRoleGroups, aadUser *msaadUser, internalUserID UserId) error {
 	nameInLogs := aadUser.profile.bestEmail()
+	m.parent.Log.Debugf("MSAAD syncRoles started for %s", nameInLogs)
```

Comment:
A part of me thinks that including this at a debug level will introduce LOADS of noise into the logs, especially if one is expecting 120something of these records every 3 minutes

I wanna recommend that we drop the log level of this one. 
```suggestion
	m.parent.Log.Tracef("MSAAD syncRoles started for %s", nameInLogs)
```